### DISCUSSION TITLE
Feature Request: Ability to Send Files (pdf, txt, etc..)

### DISCUSSION THREAD

**Discussion by hnt3b453** [balzack/databag#23] [Ideas]
I'd love the ability to send other filetypes outside of audio/video/music. Things like pdf, txt, gif, etc...

**Comment by balzack** [comment]
To clarify, would you want these properly integrated (where they would be viewable within the app as opposed to download and view outside the app) ?

**Reply by hnt3b453** [reply]
Gifs viewable within the app would be great, but for other files, downloading would be fine.

As part of this, having an easy way to set file size limits in the admin panel would also be really nice. Barring that, being able to set a file size limit in the docker compose would be an easy way to do it too. 

In the end, however it would be implmented, having an easy way to set/adjust the file size limit would be super nice.

I'd likely set mine to unlimited file size, but I'm sure others would want the ability to limit it for server management.

**Reply by ghost** [reply]
> To clarify, would you want these properly integrated (where they would be viewable within the app as opposed to download and view outside the app) ?

Hello :-) 
I really like your project and have been playing around with for the past week on web, ios and android.
For me just being able to download/save attached/sent images, videos, audio to device would be great. Being able to share it with other apps even greater.

**Reply by balzack** [reply]
The android version (published today) should have a share button that becomes visible when you tap on the message header. The ios version should be published soon (it's currently in apple's review process). Is this the functionality you are referring to? 

[edit] most importantly, thank you for using this project!

**Reply by ghost** [reply]
> Is this the functionality you are referring to? 

Yes, exactly.  
I just updated and tried it on Android and it works beautifully! Thank you!

Update: the inverse way works too ... one can share from another app to Databag
